### ISSUE TITLE
The request for training data

### ISSUE THREAD

**Issue by xiaofan-sysu** [ShuangLI59/Language-Model-Pre-training-Improves-Generalization-in-Policy-Learning#7]
Could you please provide the generated training data in the virtual home environment? I'm interested in your work and really hope to make some improvements based on this solid work!

Thank you very much!
Best Wishes!

**Comment by yanan1116** [comment]
![Image](https://github.com/user-attachments/assets/44075e2d-1f38-43a6-a5bf-0bb53381846e)

@ShuangLI59  hi, could you release the 1500 samples test set ? thanks.
